packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -113,6 +124,12 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
     setShowPeople(!showPeople);
   }, [showPeople]);
 
+  const chatProps: ChatCompositeProps = useMemo(() => {
```

Comment:
Minor: Is it necessary to use another `useMemo` for chatProps since there is already a `useMemo` for the chatAdapter on line 89? Maybe we can simply have it as:
```
const chatProps: ChatCompositeProps = { adapter: chatAdapter };
```

Reply:
That will still create a new object on each render.

We _should_ reduce to one `useMemo` though. Perhaps as:

```ts
const chatProps: ChatCompositeProps = useMemo(() => ({
    adapter: new MeetingBackedChatAdapter(meetingAdapter)
}), [meetingAdapter])
```

And then use `chatProps.adapter` wherever we need access to the adapter directly.

Reply:
okay i'll look at this!